File: app/src/main/java/com/yusril/tugaswidget/WidgetImageButton.java
package com.yusril.tugaswidget;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class WidgetImageButton extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_widget_image_button);
    }
    public void firstImageButton(View view) {
        Toast.makeText(getApplicationContext(), "Kalian telah memilih imagebutton pertama", Toast.LENGTH_LONG).show();
    }

    public void secondImageButton(View view) {
        Toast.makeText(getApplicationContext(), "Kalian telah memilih imagebutton kedua", Toast.LENGTH_LONG).show();
    }
}

File: app/src/main/java/com/yusril/tugaswidget/WidgetImageButton.kt
package com.yusril.tugaswidget

import androidx.appcompat.app.AppCompatActivity

import android.os.Bundle
import android.widget.Toast

@Suppress("DEPRECATION")
class WidgetImageButton : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_widget_image_button)
        if (supportActionBar != null) {
            supportActionBar!!.setBackgroundDrawable(resources.getDrawable(R.drawable.gradasi))
        }
    }

    fun firstImageButton() {
        Toast.makeText(applicationContext, "Kalian telah memilih imagebutton pertama", Toast.LENGTH_LONG).show()
    }

    fun secondImageButton() {
        Toast.makeText(applicationContext, "Kalian telah memilih imagebutton kedua", Toast.LENGTH_LONG).show()
    }
}
